src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -605,11 +607,11 @@ public static boolean getMobGriefingEvent(Level level, Entity entity)
         return result == Result.DEFAULT ? level.getGameRules().getBoolean(GameRules.RULE_MOBGRIEFING) : result == Result.ALLOW;
     }
 
-    public static boolean saplingGrowTree(LevelAccessor level, RandomSource randomSource, BlockPos pos)
+    public static SaplingGrowTreeEvent saplingGrowTree(LevelAccessor level, RandomSource randomSource, BlockPos pos, Holder<? extends ConfiguredFeature<?, ?>> holder)
```

Comment:
Methods must be deprecated for removal. Otherwise, removing methods is a breaking change.